.travis.yml

```diff
@@ -47,6 +47,11 @@ addons:
             - python3-gst-1.0
             - unrar
 
+before_install:
+    - sudo add-apt-repository -y "deb http://archive.ubuntu.com/ubuntu xenial main"
+    - sudo apt-get -y update
+    - sudo apt-get install sqlite3 -y
```

Comment:
Yep, that was a test, but anyway, I don't know about appveyor and I noticed it also failed, so I went back, removed that patch and instead added support for older sqlite versions, check the comment on a52ba556143cca87b561a85d448725d89e60ea0a and https://repology.org/metapackage/sqlite3/badges for a full distribution list and which sqlite3 version they have (remember, json support was introduced in 3.9.0)